pkg/chaosdaemon/pb/chaosdaemon.proto

```diff
@@ -240,3 +244,17 @@ message SetDNSServerRequest {
   bool enable = 3;
   bool enterNS = 4;
 }
+
+message InstallJVMRulesRequest {
+  string container_id = 1;
+  string rule = 2;
+  int32 port = 3;
+  bool enterNS = 4;
```

Comment:
I noticed that we always set `enterNs` to `true`. And `jvm_server.go` only works with `enterNs` being `ture`. It there any other scenarios using `false`? Or remove this field. Same situation in UninstallJVMRulesRequest.

Reply:
In chaosd, there are some similar codes. I'm planning to reuse the code in Chaos Mesh, and then need to set `enterNS` to false.